ci/pr-pipeline.yml

```diff
@@ -95,10 +96,14 @@ resources:
     source:
       url: ((scs-slack-webhook))
 
-  - name: toolsmith-env
-    type: pcf-pool
-    icon: pool
+  - name: cf-environment
+    type: shepherd
     source:
-      api_token: ((toolsmith-api-token))
-      hostname: environments.toolsmiths.cf-app.com
-      pool_name: cf-deployment
+      url: https://v2.shepherd.run
+      service-account-key: ((shepherd-service-account-key))
+      lease:
+        namespace: spring-cloud-app-broker
+        pool:
+          namespace: official
+          name: cfd
+      compatibility-mode: environments-app
```

Comment:
Please could you fix the trailing newline? We should probably have this set in `.editorconfig` if we don't already.

Reply:
Updated the `.editorconfig` it was not being picked up and had some missing things for the non-java files. Thanks!